packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1375,8 +1375,6 @@ export class DeltaManager
         }
         this.baseTerm = message.term;
 
-        this.emit("beforeOpProcessing", message);
```

Comment:
Whats a good alternative for "beforeOpProcessing"? I have been using it for measuring E2E op latency from a driver perspective (before the op actually processed by runtime)?

Reply:
the "op" event's 2nd argument is time it took to process op by runtime (+ deltamanager overhead). 
```
        const endTime = Date.now();
        this.emit("op", message, endTime - startTime);
```
Is this sufficient?

Reply:
Yes that should be sufficient for our use case.